version

```diff
@@ -1 +1 @@
-3.4.2
\ No newline at end of file
+3.4.3
```

Comment:
add the newline back to the version files

Reply:
I think I'm gonna leave it for now since that's the way it was before. Lmk if you think I should add it.